apps/marketplace/src/app/layout.tsx

```diff
@@ -34,12 +35,17 @@ const RootLayout: FunctionComponent<RootLayoutProps> = ({ children }) => {
         />
       </head>
 
-      <body style={ { backgroundColor: theme.colors.black } }>
+      <body
+        style={ {
+          backgroundColor: theme.colors.black,
```

Comment:
@scandycuz Is this just a formatting glitch between our two setups?

Reply:
No it was on a single line and the comma dangle was added with format-on-save, might have just been related to the other Format issues we've been seeing.

https://github.com/projectNEWM/newm-web/blob/dc189314b30db8171824633b0f24d38fe390bbdc/.eslintrc.json#L69-L70